txserver/structures.go

```diff
@@ -1,11 +1,11 @@
 package main
 
 type Command struct {
-	Command  string      `json:"Command"`
-	Username string      `json:"Username"`
-	Amount   interface{} `json:"Amount"`
-	Stock    string      `json:"Stock"`
-	Filename string      `json:"Filename"`
+	Command  string  `json:"Command"`
+	Username string  `json:"Username"`
+	Amount   float32 `json:"Amount"`
```

Comment:
I have changed this for now. I checked with TA and he said that we don't need to log anything if the sent command has invalid properties.

Reply:
![image](https://user-images.githubusercontent.com/39280329/155829823-83939e28-d7a4-437a-bb80-76aaed8fd6fa.png)